devices/xiaomi.js

```diff
@@ -745,7 +745,9 @@ module.exports = [
         fromZigbee: [fz.xiaomi_contact, fz.ias_contact_alarm_1, fz.aqara_opple],
         toZigbee: [],
         meta: {battery: {voltageToPercentage: '3V_2850_3000'}},
-        exposes: [e.contact(), e.battery(), e.battery_voltage()],
+        exposes: [e.contact(), e.battery(), e.battery_voltage(),
+            exposes.enum('battery_cover', ea.STATE, ['CLOSE', 'OPEN']).withDescription('Battery cover position'),
```

Comment:
Sorry for not having noticed this on my first review. But this can become a binary:

```
exposes.binary('battery_cover', ea.STATE, 'OPEN', 'CLOSE')
```

Reply:
Done. Now it's binary.